accessibility-oscilloscope/overlay/etc/init.d/S16oscope

```diff
@@ -21,7 +21,15 @@ enable-haptic-gpio
 
 # start programs in background
 
-pipe-wrapper "signal-compiler /dev/stdin /dev/stdout" /run/sigcomp-in /run/pnt-lut-pgm &
+# "left side" of tree
+
+pipe-wrapper "trigger-data -i /dev/stdin -o /dev/stdout" $OSCOPE_FIFO $SIGPROC_FIFO &
+
+pipe-wrapper "sigproc -i /dev/stdin -o /dev/stdout" $SIGPROC_FIFO $SIGCOMP_FIFO &
+
+pipe-wrapper "signal-compiler /dev/stdin /dev/stdout" $SIGCOMP_FIFO $PGM_FIFO &
+
+wacom-btn > $OSCOPE_FIFO &
```

Comment:
i wonder if we're having a problem with program ordering here with the reader end being open. we ran into this before, thoughts @cnorthway ?